test/LaTeXImportTests.js

```diff
@@ -0,0 +1,297 @@
+            });
+        });
+        describe('Division', () => {
+            it('\\frac', () => {
+                assert.equal('3', nerdamer.convertFromLaTeX('\\frac{6}{2}').toTeX());
+            });
+            it('Division Symbol', () => {
+                assert.equal('3', nerdamer.convertFromLaTeX('6 \\div 2').toTeX());
+            });
+            it('Slash', () => {
+                assert.equal('2', nerdamer.convertFromLaTeX('6/3').toTeX());
+            });
+            it('Nested fractions', () => {
+                assert.equal('1', nerdamer.convertFromLaTeX('\\frac{\\frac{\\frac{7}{4}}{2\\frac{7}{3}}}{\\frac{6}{16}}').toTeX());
+            });
+            it('Display Fraction', () => {
+                assert.equal('3', nerdamer.convertFromLaTeX('\\dfrac{6}{2}').toTeX());
+            });
+        });
+        it('Exponents', () => {
+            assert.equal('8', nerdamer.convertFromLaTeX('2^3').toTeX());
+            assert.equal('8', nerdamer.convertFromLaTeX('2^{3}').toTeX());
+            assert.equal('8', nerdamer.convertFromLaTeX('{2}^{3}').toTeX());
+        });
+    });
+
+    context('Brackets', () => {
+        describe('Parenthesies', () => {
+            it('Simple Brackets', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2(1+1)').toTeX());
+            });
+            it('\\left and \\right', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\left(1+1\\right)').toTeX());
+            });
+            it('\\big', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\big(1+1\\big)').toTeX());
+            });
+            it('\\bigg', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\bigg(1+1\\bigg)').toTeX());
+            });
+            it('\\Big', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\Big(1+1\\Big)').toTeX());
+            });
+            it('\\Bigg', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\Bigg(1+1\\Bigg)').toTeX());
+            });
+        });
+        describe('Square Brackets', () => {
+            it('Simple Brackets', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2[1+1]').toTeX());
+            });
+            it('\\left and \\right', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\left[1+1\\right]').toTeX());
+            });
+            it('\\big', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\big[1+1\\big]').toTeX());
+            });
+            it('\\bigg', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\bigg[1+1\\bigg]').toTeX());
+            });
+            it('\\Big', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\Big[1+1\\Big]').toTeX());
+            });
+            it('\\Bigg', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\Bigg[1+1\\Bigg]').toTeX());
```

Comment:
Square brackets indicate vectors, so they should be treated as such.
```suggestion
                assert.equal('\\left[4\\right]', nerdamer.convertFromLaTeX('2 \\Bigg[1+1\\Bigg]').toTeX());
```